roboto/api_types.py

```diff
@@ -14,6 +16,31 @@
 FileID = NewType('FileID', str)
 
 
+class FileDescription(NamedTuple):
+    """Describe an in-memory file to be sent through the API."""
+
+    binary_source: Union[BinaryIO, bytes]
+    basename: str
+    mime_type: str = 'application/octet-stream'
+
+
+# pylint: disable=invalid-triple-quote
+InputFile = Union[Path, BufferedIOBase, FileID, FileDescription, URL]
+"""An InputFile can be either:
+
+A Path object: This will be used to open the file and read it to send.
+A BufferedIOBase object: This supports sending an already open file.
+A FileID from Telegram's Bot API itself: This will just send a previously sent file.
+A FileDescription object: This is used to send binary data directly as a file, or to
+                          customize mimetype or filename.
+A URL: This instructs Telegram's server to download the file and send it. In
+       this case, file size is severely limited.
+
+Refer to https://core.telegram.org/bots/api#sending-files for more information.
+"""
```

Comment:
Python itself won't recognize it, because there is no such thing as a variable docstring. But tools such as Sphinx, which I plan to use eventually to extract the API reference do. About VSCode, I have no idea, but it sure could parse it the same as Sphinx does.